### DISCUSSION TITLE
Alert on 0 events in the past hour

### DISCUSSION THREAD

**Discussion by adamladd17** [jertel/elastalert2#882] [Q&A] [answered]
Is there a way to alert on a frequency of size 0? Basically we only want to send out alerts if we have received no events over the past hour.

**Comment by jertel** [comment, marked as answer]
The [flatline](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#flatline) rule type might be a better fit for you.

**Reply by adamladd17** [reply]
Thank you I completely missed that one! That is exactly what I am looking for.
